resources/views/docs/1/the-basics/screens.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Screens
+order: 850
+---
+# Screens
+
+The `Screen` facade lets you get information about the screens currently connected to the computer.
+
+```php
+use Native\Laravel\Facades\Screen;
+```
+
+## Displays
+
+The `displays` method gives you an array of information about all the displays actually used.
+
+The Display object represents a physical display connected to the system. A fake display may exist on a headless system, or a display may correspond to a remote, virtual display. If you use an external display with your laptop screen closed, the internal screen of your laptop will not be part of the array.
+
+See [Display object](https://www.electronjs.org/docs/latest/api/structures/display) documentation.
```

Comment:
You can leave this here for now, but I'm wary of linking out to external docs for a couple of reasons:

1. They may change in ways that we're not aware of
2. It becomes more complicated as we move to supporting more drivers, e.g. Tauri

Reply:
Yes, I thought of that, but since in this current version we basically send the same exact structure as Electron without touching it, I thought it would be a more accurate explanation for each element of the array.

For Tauri, I imagine that it will not come before 2.0 and that we could change the docs when the time comes.

Reply:
Also, I didn't see any JavaScript or Rust API for screens, even in 2.0 RC.

Reply:
Oh, I see, the core is already spliced into different plugins. It doesn't seem to have this plugin yet. I did see a plugin to remember the last position of the windows, which is good.